### ISSUE TITLE
Next version 9.5.3 introduces breaking changes to _app.tsx when using Yarn

### ISSUE THREAD

**Issue by DopamineDriven** [vercel/next.js#16966]
# Bug report

## Describe the bug
Next version 9.5.3 breaks when used with yarn. I remade a repo using npm instead and it was smooth sailing.
```
error - ./pages/_app.tsx
Error: C:\Users\Anthr\asross-portfolio\pages\_app.tsx: Invalid page config export found. Expected object but got import in file \pages\_app.tsx. See: https://err.sh/vercel/next.js/invalid-page-config 
event - build page: /next/dist/pages/_error
wait  - compiling...
error - ./pages/_app.tsx
Error: C:\Users\Anthr\asross-portfolio\pages\_app.tsx: Invalid page config export found. Expected object but got import in file \pages\_app.tsx. See: https://err.sh/vercel/next.js/invalid-page-config 
Error: Cannot find module 'C:\Users\Anthr\asross-portfolio\.next\build-manifest.json'
Require stack:
- C:\Users\Anthr\asross-portfolio\node_modules\next\dist\next-server\server\load-components.js
- C:\Users\Anthr\asross-portfolio\node_modules\next\dist\next-server\server\api-utils.js
- C:\Users\Anthr\asross-portfolio\node_modules\next\dist\next-server\server\next-server.js
- C:\Users\Anthr\asross-portfolio\node_modules\next\dist\server\next.js
- C:\Users\Anthr\asross-portfolio\node_modules\next\dist\server\lib\start-server.js
- C:\Users\Anthr\asross-portfolio\node_modules\next\dist\cli\next-dev.js
- C:\Users\Anthr\asross-portfolio\node_modules\next\dist\bin\next
```

"Invalid page config export found. Expected object but got import in file pages\ _app.tsx."

## To Reproduce
Create a next project using yarn. In _app.tsx, use the following code
```tsx
import 'styles/index.css';
import '@tailwindcss/typography';
import { AppProps } from 'next/app';
import { ReactElement } from 'react';

export function reportWebVitals(metric: any): void {
	console.log(metric);
}

function App({ Component, pageProps }: AppProps): ReactElement {
	return <Component {...pageProps} />;
}

export default App;
```

Steps to reproduce the behavior, please provide code snippets or a repository:

Go to a next typescript repository that uses yarn instead of npm. update your next version to 9.5.3
Launch `yarn dev`. It will throw an event `/next/dist/pages/_error`
`Cannot find module ...build-manifest.json`

## Expected behavior

Expected _app.tsx to not unforeseeably error out. 

## Screenshots

Launching with next version 9.5.3
![image](https://user-images.githubusercontent.com/46355797/92622674-f0591500-f28a-11ea-91b9-0f770e406363.png)
Rolling the version back to next version 9.5.2
![image](https://user-images.githubusercontent.com/46355797/92622747-0961c600-f28b-11ea-9181-ecb0aef2dbfa.png)
Next launches successfully on `yarn dev` with next 9.5.2
![image](https://user-images.githubusercontent.com/46355797/92622786-15e61e80-f28b-11ea-89e3-3c6393d28050.png)

## System information

- OS: Windows
- Version of Next.js: 9.5.3 introduces breaking changes
- Version of Node.js: 14.10

## Additional context

Rollback your next version to 9.5.2 to resolve the error if using typescript and yarn with next until the breaking changes are resolved by the vercel/next team.

**Comment by laugharn** [comment]
I believe this was fixed in 9.5.4-canary.4: https://github.com/vercel/next.js/pull/16834

**Comment by DopamineDriven** [comment]
@laugharn you are correct. I just ran `yarn add next@9.5.4` and that resolved the issue as well. When running `yarn add next` without targeting a version explicitly it installs next 9.5.3. Thanks for the prompt response!

**Comment by timneutkens** [comment]
Fixed in #16834 indeed.

**Comment by balazsorban44** [comment]
This issue has been automatically locked due to no recent activity. If you are running into a similar issue, please create a new issue with the steps to reproduce. Thank you.
